libraries/rush-lib/src/api/test/LastInstallFlag.test.ts

```diff
@@ -105,19 +105,4 @@ describe(LastInstallFlag.name, () => {
     await expect(flag2.checkValidAndReportStoreIssuesAsync({ rushVerb: 'install' })).resolves.not.toThrow();
     await expect(flag2.checkValidAndReportStoreIssuesAsync({ rushVerb: 'install' })).resolves.toEqual(false);
   });
-
-  it("ignores a specified option that doesn't match", async () => {
```

Comment:
This functionality no longer behaves the same way?

Reply:
As the LastInstallFlag no longer accepts options that aren't specified in the spec, so it no longer makes sense to have a test that checks if unspecified options are ignored